doc/nvim-tree-lua.txt

```diff
@@ -240,6 +241,8 @@ Subsequent calls to setup will replace the previous configuration.
           git_placement = "before",
           padding = " ",
           symlink_arrow = " ➛ ",
+          modified = "[+]",
```

Comment:
We should use a single character `glyphs.modified`

There is code in the renderer that assumes icons are single characters.